code/js/rcalc-keypress.js

```diff
@@ -0,0 +1,107 @@
+var isSecondPressed = false;
+var collectionOfKeys = [];
+var nextToLastCharacter;
+
+document.addEventListener("keyup", function(event) {
+  if (event.defaultPrevented) {
+    return;
+  }
+  var key = event.key;
+  collectionOfKeys.push(key);
+  nextToLastCharacter = collectionOfKeys[collectionOfKeys.length - 2];
```

Comment:
Why minus -2? Specify or put comment. Magic number man.

Reply:
explanation of why -2 in the comment